pcsx2/GS/GSState.cpp

```diff
@@ -3070,6 +3069,12 @@ __forceinline void GSState::HandleAutoFlush()
 			tex_rect.w = std::max(tex_rect.w, tex_coord.y);
 		}
 
+		// If the draw was 1 line thick, make it larger as rects are exclusive of ends.
+		if (tex_rect.x == tex_rect.z)
+			tex_rect += GSVector4i(0, 0, 1, 0);
```

Comment:
Not sure if the compiler will automatically do this, but using `GSVector4i::cxpr(...)` will ensure it's a constant loaded from memory, rather than constructed in-place.